src/search-filters/components/Bookmark.js

```diff
@@ -0,0 +1,41 @@
+import React, { PureComponent } from 'react'
+// import styles from './Bookmark.module.css'
+import { Checkbox } from 'src/common-ui/components'
+
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { actions, selectors } from '../'
+
+class Bookmark extends PureComponent {
+    static propTypes = {
+        bookmarkFilter: PropTypes.bool.isRequired,
+        onShowOnlyBookmarksChange: PropTypes.func.isRequired,
+    }
+
+    render() {
+        return (
+            <div>
```

Comment:
Use `React.Fragment` instead of `div` if the only reason for it is to satisfy react's requirement of having a single parent element per JSX expression.